README.md

```diff
@@ -154,11 +161,8 @@ Custom configuration for the Celery workers are listed below:
   to the main Cachito repositories (e.g. `cachito-js`). This is needed if the Nexus instance that
   hosts the main Cachito repositories has anonymous access disabled. This is the case if Cachito
   utilizes just a single Nexus instance.
-* `cachito_nexus_npm_proxy_repo_url` - the URL to the `cachito-js` repository which is a Nexus group
-  that points to the `cachito-js-hosted` hosted repository and the `cachito-js-proxy` proxy
-  repository. This defaults to `http://localhost:8081/repository/cachito-js/`. This only needs to
-  change if you are using the two Nexus instance approach as described in the "Nexus For npm"
-  section or you use a different name for the repository.
+* `cachito_nexus_request_repo_prefix` - the prefix of Nexus proxy repositories made for each
+  request for applicable package managers (e.g. `cachito-js-1`). This defaults to `cachito-`.
```

Comment:
`js` is not a package manager. Should this be `npm`? Or maybe we need to change "package manager" with "content type"?

Reply:
To summarize offline conversation. Yes, this should be done, but it's a larger change. Let's skip it for now and address it as a technical debt issue in the future if there's a need for it.